### DISCUSSION TITLE
Suggest Answer button in Discussions section

### DISCUSSION THREAD

**Discussion by samarthsubramanya** [community/community#7385] [Mobile] [answered]
The Suggest answer button in the Discussions section is not situated at the exact end. As a result the button floats a certain dp above the end due to which a portion of background text appears below the button too.
Found in: Android App
Version: 1.39.0
 ![Screenshot_20211106-090256_GitHub.png](https://user-images.githubusercontent.com/35397647/140596695-9636809a-8905-4daf-81e4-6322f2c48446.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @JohnX4321 — I agree this isn't a great experience, and doesn't feel very clear. I'll pass this along to our team to figure out how to make this better.

**Comment by ZuOliver** [comment]
Ola esse aplicativo é de desenvolvedores?@integra@loja@uber
